File: app/src/at-trivo/java/asiantech/internship/summer/restful/model/Image.java
package asiantech.internship.summer.restful.model;

import com.google.gson.annotations.SerializedName;

public class Image {
    @SerializedName("image_id")
    public String imageId;
    @SerializedName("permalink_url")
    public String permalinkUrl;
    @SerializedName("thumb_url")
    public String thumbUrl;
    public String url;
    public String type;
}

File: app/src/at-trivo/java/asiantech/internship/summer/restfulkotlin/model/Image.kt
package asiantech.internship.summer.restfulkotlin.model

import com.google.gson.annotations.SerializedName

open class Image {
    @SerializedName("image_id")
    var imageId: String? = null
    @SerializedName("permalink_url")
    var permalinkUrl: String? = null
    @SerializedName("thumb_url")
    var thumbUrl: String? = null
    var url: String? = null
    var type: String? = null
}